### PR TITLE
Overview styling basics

### PR DESCRIPTION
**Pull request by allienworks**

### CODE REVIEW

File: src/assets/theme.scss [hunk 1]
```diff
@@ -305,18 +328,18 @@ input[type="text"] {
...
-  .mat-select-trigger{height: 37px;}
+  .mat-select-trigger{ height: 37px; }
 }
 
 .header-nav {
-  list-style-type: none;padding: 0;
+  list-style-type: none; padding: 0;
   li {
-    font-weight: bold;
+    font-size: 16px; font-weight: normal; text-transform: uppercase; color: $text-muted;
```
Comment (rynomster) [inline src/assets/theme.scss]: long single line css?

File: src/app/wallet/balances/balance.component.scss [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+        color: #222728;
+        &.big { font-size: 30px; }
+        &.small { font-size: 20px; }
+    }
+    .point {
+        color: #02E8B0;
+        font-size: 35px;
+    }
+    .description {
+        color: $text-muted; text-transform: uppercase;
```
Comment (rynomster) [inline src/app/wallet/balances/balance.component.scss]: multiple lines

File: src/app/overview/overview.component.scss [hunk 3]
```diff
@@ -1,14 +1,59 @@
...
+            position: relative; top: 9px;
+            width: auto; height: auto; margin: -11px 15px 0 0;
+        }
+        .title {
+            display: inline-block;
+            text-transform: uppercase; font-weight: bold;
+            & > .enabled { color: $color; }
+            & > .not-enabled { color: $color-alert; }
+        }
+        button { font-size: 12px; text-transform: uppercase; }
```
Comment (rynomster) [inline src/app/overview/overview.component.scss]: Please use multiple lines

File: src/app/overview/overview.component.scss [hunk 4]
```diff
@@ -1,14 +1,59 @@
...
+md-card.multiple {
+    margin-bottom: 20px;
+}
+
+.stat-box {
+    & > .value {
+        font-size: 24px;
+    }
+    & > .description {
+        color: $text-muted; text-transform: uppercase;
```
Comment (rynomster) [inline src/app/overview/overview.component.scss]: multiple lines

File: src/app/overview/overview.component.scss [hunk 5]
```diff
@@ -1,14 +1,59 @@
...
+        margin-top: 15px;
+    }
+}
+
+.cold-staking {
+    .staking-node {
+        padding: 24px 24px 30px;
+        .cs-icon { // snowflake
+            font-size: 40px; color: $bg-shadow;
```
Comment (rynomster) [inline src/app/overview/overview.component.scss]: multiple lines

File: src/app/app.component.scss [hunk 6]
```diff
@@ -1,17 +1,50 @@
...
+  }
+}
+
+.mat-icon { // all icons in menu
+  margin: 12px 6px 0 0 !important;
+  color: #4B4A4F !important;
+}
+
+.menu-item {
+  font-family: $font; font-size: 11.5px; font-weight: 600; color: white;
```
Comment (rynomster) [inline src/app/app.component.scss]: multiple lines

File: src/app/app.component.scss [hunk 7]
```diff
@@ -1,17 +1,50 @@
...
+}
+
+.mat-divider { // section divider
+  border-bottom: 1px solid #333; border-top: 1px solid #15191A;
```
Comment (rynomster) [inline src/app/app.component.scss]: multiple lines

File: src/app/app.component.scss [hunk 8]
```diff
@@ -1,17 +1,50 @@
...
   position: relative;
   top: 50%;
   transform: translateY(-50%);
 }
 
+.mat-expansion-panel { // accordion submenu
+  border-bottom: 1px solid #15191A;
+  box-shadow: 0 1px 0 #333 !important;
+  .mat-expansion-panel-header {
+    font-family: $font; font-size: 11.5px; color: #646869;
```
Comment (rynomster) [inline src/app/app.component.scss]: multiple lines

File: src/app/app.component.scss [hunk 9]
```diff
@@ -1,17 +1,50 @@
+$font: "Open Sans", sans-serif;
```
Comment (rynomster) [inline src/app/app.component.scss]: We can move these to helper.scss ?

Comment (allienworks) [reply src/app/app.component.scss]: Doesn't work when you put it out of the component's context.. the variables don't get shared across files.

File: src/app/app.component.scss (hunk 8 above)
Comment (allienworks) [reply src/app/app.component.scss]: fixed ✓

File: src/app/app.component.scss (hunk 6 above)
Comment (allienworks) [reply src/app/app.component.scss]: fixed ✓

File: src/app/app.component.scss (hunk 7 above)
Comment (allienworks) [reply src/app/app.component.scss]: fixed ✓

File: src/app/overview/overview.component.scss (hunk 4 above)
Comment (allienworks) [reply src/app/overview/overview.component.scss]: fixed ✓

File: src/app/overview/overview.component.scss (hunk 5 above)
Comment (allienworks) [reply src/app/overview/overview.component.scss]: fixed ✓

File: src/app/overview/overview.component.scss (hunk 3 above)
Comment (allienworks) [reply src/app/overview/overview.component.scss]: fixed ✓

File: src/app/wallet/balances/balance.component.scss (hunk 2 above)
Comment (allienworks) [reply src/app/wallet/balances/balance.component.scss]: fixed ✓

File: src/assets/theme.scss (hunk 1 above)
Comment (allienworks) [reply src/assets/theme.scss]: fixed ✓

File: src/app/app.component.scss (hunk 9 above)
Comment (rynomster) [reply src/app/app.component.scss]: Oh right, you have to import it to use the variables in another file.. We should make a common scss file that we can import :)

**Pull request merged**